build-logic/commons/src/main/kotlin/bisq.java-conventions.gradle.kts

```diff
@@ -1,3 +1,5 @@
+import gradle.kotlin.dsl.accessors._a0d7d695c10041dcc5b883f9bcd31010.implementation
```

Comment:
Is this a standard package path? The '_a0d7d695c10041dcc5b883f9bcd31010' looks like its generated dynamically...

Reply:
that was auto-added by Intllij... doesn't sound like its really needed I'll erase this as well after a quick test